KitchenLib/src/Patches/DebugLogPatch.cs

```diff
@@ -35,7 +38,27 @@ public KLLogHandler(ILogHandler logHandler)
 
 			public void LogException(Exception exception, UnityEngine.Object context)
 			{
-				logHandler.LogException(exception, context);
+				string MOD_NAME = "PlateUp!";
```

Comment:
This should probably be a static readonly const instead of defined in this method

Reply:
@Yariazen - This is for you

Reply:
Just hardcoded the PlateUp! case because it should not ever be anything else and no other classes should be referencing it.